examples/process-persistence/README.md

```diff
@@ -230,19 +228,28 @@ postgresql database configuration we can run the following command
 mvn clean package -Pcontainer,postgresql
 ```
 This will build this example's Quarkus application with the corresponding database configuration and create a Docker 
-image that will be used in the `docker-compose` template.
+image that will be used in the `docker compose` template.
 
-To execute the full example, run the following command inside the `docker-compose` folder
+Finally, to start the example using `docker compose`, run
 
-```shell
-# cd docker-compose
-sh startContainers.sh <dbtype>
 ```
-If we don't specify a dbtype `sh startContainers.sh`, database configurations of postgresql is used by default in 
-container mode.
+docker compose -f ./docker-compose/docker-compose-<dbtype>.yml up
+```
+
+For e.g. to start the example with postgresql run
+
+```bash
+docker compose -f ./docker-compose/docker-compose-postgresql.yml up
+```
+
+To stop and remove containers run
+
+```
+docker compose -f ./docker-compose/docker-compose-<dbtype>.yml down
+```
 
 > **_IMPORTANT:_** if you are running this example on macOS and you are not using **Docker Desktop**, please append
```

Comment:
I think that this section is no longer required, at least if we are fixing the urls to point to `http://127.0.0.1:8080` this `kubernetes.docker.internal` won't be used anymore in the docker-compose files.